### DISCUSSION TITLE
How to detect WLAN environments that do not support nicegui

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [zauberzeug/nicegui#1442] [Q&A]
### Question

I wrote about #960 

_It works locally if i add "localhost" to browser permissions. Does not work as expected e.g. on http://nicesrt.bitplan.com/ so unfortunately the github link there also doesn't work: https://github.com/WolfgangFahl/nicesrt_

but the reason for "it does not work" seems to be hotel WLAN that somehow makes nicegui not behave properly. If i switch on my VPN things run slower but fine.

There might be e.g. a blocked websocket port or similiar "feature". How can the proper operation of nicegui be checked under such circumstances? Do we have some kind of connection check that e.g. can be displayed with an led like button to show that nicegui's seb socket connection is alive?

**Comment by rodja** [comment]
What would be the kind of configuration you need to do on a network to disallow websockets? I was under the impression that it tunnels fine over http...

**Comment by WolfgangFahl** [comment]
I have no clue what was done to cripple the network that nicegui didn't work -it's just an assumption how it happened.

**Comment by rodja** [comment]
It would be interesting to find a reproduction (without staying in the hotel where @WolfgangFahl experienced the problem).
